src/pyramid_debugtoolbar/panels/request_vars.py

```diff
@@ -124,9 +124,9 @@ def __init__(self, request):
             _post_preview = None
             try:
                 try:
-                    _post_converted = request.body.decode('utf-8')
+                    _post_converted = request.text
                 except:
-                    _post_converted = "[ could not decode POST body to UTF-8 ]"
+                    _post_converted = "[ could not decode POST body]"
```

Comment:
I think this should just say `[... {request.content_type}; {request.content_length} bytes ...]`

Reply:
I should have used `request.content_length` instead of calculating the size too.

how does this look:

```
                    _post_converted = "[... %s; %s bytes ...]" % (
                        request.content_type,
                        request.content_length,
                    )
```

Reply:
It's fine, I decided against the `;` because that's actual syntax in a mime type so you can see my updated suggestion.

My only other feedback on this PR is that I'd appreciate if you stuck to code conventions that are already in the project... your egregious whitespace and `#---` lines and annotating the imports as stdlib/local/etc and importing `.` before `absolute` imports are all things that need to get sorted out. I will however capitulate and do it for you this once if you want me to merge it and clean it up - but in future PRs please don't add your own niceties to existing code bases unless it's done everywhere... no individual files are special. I recognize we haven't setup the black/isort stuff in this project yet so I'm trying to be a bit lenient on it and offering to help out if you're confused about which things I'm complaining about.

Reply:
Sorry, about the "#---" delimiters and docs on the imports; this was pulled from internal systems that use those conventions.  I'll remove them in the next commit.

This is running against black though, so all the whitespace is from that.

Reply:
also, no worries or offense. I'm sorry for deviating from the style.  let me know what black is doing wrong on the next commit.

Reply:
Ack! I'm sorry the existing "request_vars.py" got picked up by black in the last commit.  I can redo that file to not be formatted with it so the changeset is smaller.  It was only supposed to be running on the new files; I must have accidentally run it somehow.